### PR TITLE
Add null and ignore_missing check to handle event.original field.

### PR DESCRIPTION
**Pull request by ritalwar**
## Proposed commit message

This PR add null and ignore_missing check to handle event.original field in multiple packages which are as follows:

- ActiveMQ
- CoreDns
- IBMMQ
- MongoDB
- Salesforce
- Spring_Boot
- vSphere

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #7822

### CODE REVIEW

File: packages/activemq/data_stream/audit/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ processors:
       field: event.original
       value: "{{{message}}}"
       ignore_empty_value: true
```
Comment (muthu-mps) [inline packages/activemq/data_stream/audit/elasticsearch/ingest_pipeline/default.yml:17]: @ritalwar - As we have the `ignore_missing` check in most places. Can we adopt the same option here as well and replace the `ignore_empty_value` with `ignore_missing` option?

Comment (ritalwar) [reply packages/activemq/data_stream/audit/elasticsearch/ingest_pipeline/default.yml:17]: @muthu-mps In the Rename processor, we use `ignore_missing`, while in the Set processor, `ignore_empty_value` serves the same purpose.

Comment (ritalwar) [reply packages/activemq/data_stream/audit/elasticsearch/ingest_pipeline/default.yml:17]: The only difference between SET and RENAME in this scenario is that with SET, we end up having two fields, namely message and event.original. Ideally, we should eliminate the message field after renaming. However, it should apply to other renamed fields as well, and the removal of the message field is not being addressed as part of this PR.

**Comment by ishleenk17** [comment]
@ritalwar : In some plavces there were set processors, in some rename. Whats the general guidance you have followed here ?

**Comment by ritalwar** [comment]
> @ritalwar : In some plavces there were set processors, in some rename. Whats the general guidance you have followed here ?

I haven't modified any processors; I've maintained the set and rename operations as they were and simply introduced these checks.

**Comment by ishleenk17** [comment]
> > @ritalwar : In some plavces there were set processors, in some rename. Whats the general guidance you have followed here ?
> 
> I haven't modified any processors; I've maintained the set and rename operations as they were and simply introduced these checks.

Ok, looks good!

**Review by ishleenk17 — APPROVED** [review]

**Pull request merged**